QEfficient/generation/text_generation_inference.py

```diff
@@ -144,15 +144,17 @@ def cloud_ai_100_exec_kv_helper(
     enable_debug_logs: bool = False,
     stream: bool = True,
     write_io_dir: Optional[str] = None,
+    session=None,
```

Comment:
why is this being added?

Reply:
Can't we change utils to pass `qpc_path` instead of `session`?

Reply:
If we pass qpc_path then we need to create session inside run_kv_model_on_cloud_ai_100 function in run_util.py, for which we need to import QAICInferenceSession, When running tests it will fail during this import if machine does not have aic card.

Reply:
Don't we skip the ai 100 execution test anyway when device is not present?

Reply:
If machine does not have aic card, it fails in get_available_device_id in device_utils.py, sorry for the confusion, changed it to take qpc path instead of session.